pkg/ops/transferOps.go

```diff
@@ -75,16 +75,13 @@ func Approve(ctx context.Context, logger log.Logger, _spender common.Address, am
 	return nil
 }
 
-func Balance(ctx context.Context, addr common.Address) error {
-	client := ctx.Value(tellorCommon.ClientContextKey).(rpc.ETHClient)
-
+func Balance(ctx context.Context, client rpc.ETHClient, getterInstance *getter.TellorGetters, addr common.Address) error {
+	//client.BalanceAt doesn't need the context, will remove later since it's used elsewhere
```

Comment:
The `ctx context.Context` in  `client.BalanceAt(context.Background()` is used properly. It is for cancelling a request, This is the main purpose of a context - you can cancel it manually - for example if you want to exit the application(by calling context cancel) without waiting for a reply or can set it to auto-cancel for example if you want to set a timeout of lets say 1sec.

Reply:
in our case we will use the context to be able to stop pending eth client requests when a SIGTERM is sent to the app.